spec/features/manage_cmr/groups/groups_list_spec.rb

```diff
@@ -237,7 +236,7 @@
 
```

Comment:
Looking at the comment on line 191, it seems like maybe we should remove some of the providers in the login line to ensure that the modification for admin users that don't have all the providers still show groups from all the providers. What do you think?

Reply:
Yes, I removed 3 of the providers and updated the context description.